app/javascript/components/GuitarInfo.js

```diff
@@ -20,18 +69,90 @@ const GuitarInfo = ({ name, url, price, description }) => {
           </GridItem>
           <GridItem>
             <CardTitle>{name}</CardTitle>
-            <CardBody>Price: {price}</CardBody>
-            <CardFooter>Description: {description}</CardFooter>
+            <CardBody>
+              Price: {price} <br />
+              Description: {description}
+            </CardBody>
+            <CardFooter>
+              <Button variant="secondary" onClick={editGuitar}>
+                Edit
+              </Button>
+            </CardFooter>
           </GridItem>
         </Grid>
       </Card>
+
+      {isFormModalOpen && (
```

Comment:
`isFormModalOpen` is used here to keep the DOM from being over packed with elements
(each GuitarInfo had a Modal as a subcomponent, while now only the one FormModal who is open would be loaded to the DOM)
Thank you @MariaAga for mentioning it 🙏🏼 
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator